executor/executor.cc

```diff
@@ -1308,6 +1309,15 @@ uint32* write_output(uint32 v)
 	return output_pos++;
 }
 
+uint32* write_output_64(uint64 v)
+{
+	if (output_pos < output_data || (char*)output_pos >= (char*)output_data + kMaxOutput)
```

Comment:
We now need some adjustment to the check I think. It assumes we write a single element.

Reply:
good finding, thanks

Reply:
done